### DISCUSSION TITLE
Warning: Array to string conversion

### DISCUSSION THREAD

**Discussion by mswirydowicz** [bootscore/bootscore#779] [Q&A]
Hi,

I have problems with this communicats:

Warning: Array to string conversion in C:\Users\mswir\Local Sites\test-new\app\public\wp-content\themes\bootscore\inc\scssphp\src\Compiler.php on line 927

Warning: Array to string conversion in C:\Users\mswir\Local Sites\test-new\app\public\wp-content\themes\bootscore\inc\scssphp\src\Compiler.php on line 927

Warning: Array to string conversion in C:\Users\mswir\Local Sites\test-new\app\public\wp-content\themes\bootscore\inc\scssphp\src\Compiler.php on line 927

Any ideas?

**Comment by crftwrk** [comment]
Please share a link to your site. You are on your local machine and it seems that this one doesn't like the compiler. Transfer you project to a public server and issue should be solved by itself.

**Comment by darrinbreen** [comment]
I'm getting the same warning on a public server? https://dev-rcb.pantheonsite.io

**Reply by crftwrk** [reply]
We will fix this in the next release, it is a bug in https://github.com/scssphp/scssphp/issues/606. Please follow issue https://github.com/bootscore/bootscore/issues/801.

Until this is fixed:

If `define( 'WP_DEBUG', true );` is used in `wp-config.php`, set it to `false` and ignore it.

or

If no WooCommerce is active, deactivate Woo styles in child's `assets/scss/main.scss`:

```scss
@import "bootscore-woocommerce"; // Comment or delete this line if WooCommerce styles are not needed
```
